gitmap.go

```diff
@@ -109,6 +108,7 @@ func Map(repository, revision string) (*GitRepo, error) {
 }
 
 func git(args ...string) ([]byte, error) {
+	fmt.Println("call git", args)
```

Comment:
Can you remove this?

Reply:
Sure, was for "debugging" only. See a26f38e067735f5fb393bbfa3f5a183090e264e4